traffic_ops/traffic_ops_golang/login/login.go

```diff
@@ -63,7 +63,9 @@ WHERE name='tm.instance_name' AND
 `
 const userQueryByEmail = `SELECT EXISTS(SELECT * FROM tm_user WHERE email=$1)`
 const setTokenQuery = `UPDATE tm_user SET token=$1 WHERE email=$2`
-const UpdateLoginTimeQuery = `UPDATE tm_user SET last_authenticated = now() WHERE username=$1`
+
+// UpdateLoginTimeQuery is meant to only update the last_authenticated field once per minute in order to avoid row-locking when the same user logs in frequently.
+const UpdateLoginTimeQuery = `UPDATE tm_user SET last_authenticated = now() WHERE username=$1 AND last_authenticated < now() - INTERVAL '1 minute'`
```

Comment:
Since the other SQL keywords are uppercase, should `now()` be uppercase? Should `minute`?